fapi-client/src/main/scala/com/gu/facia/api/contentapi/ContentApi.scala

```diff
@@ -24,21 +24,16 @@ object ContentApi extends StrictLogging {
   type AdjustSearchQuery = SearchQuery => SearchQuery
   type AdjustItemQuery = ItemQuery => ItemQuery
 
-  def buildHydrateQueries(client: ContentApiClientLogic, ids: List[String], adjustSearchQuery: AdjustSearchQuery = identity): Response[Seq[SearchQuery]] = {
-    def queryForIds(ids: Seq[String]) = adjustSearchQuery(client.search
-      .ids(ids mkString ",")
-      .pageSize(ids.size)
-      .showFields("internalPageCode"))
-
+  def buildHydrateQueries(client: ContentApiClient, ids: List[String], adjustSearchQuery: AdjustSearchQuery = identity, itemQueries: ItemQueries = ItemQueries): Response[Seq[SearchQuery]] = {
```

Comment:
Why pass in `ItemQueries` as a parameter?

Reply:
Because it has to be mocked in the tests 😩